__tests__/user/user.query.test.js

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-client');
+
+require('dotenv').config();
+
+let token, userId
+
+describe('queries', () => {
+    beforeAll(async () => {
+        const register = await client.mutate({
+            mutation: gql`
+                mutation {       
+                    registerUser(user: {
+                        firstName: "Test" 
+                        lastName: "User"
+                        email: "test.email@gmail.com"
+                        password: "12345678Te"
+                    }) {
+                        _id
+                        firstName
+                        lastName
+                        email
+                        role
+                        registrationDate
+                        credentials {
+                            tokenPass
+                        } 
+                    }
+                }
+            `
+        })
+
+        userId = register.data.registerUser._id
+
```

Comment:
Hey, you can also add here update mutation to populate user with all its fields to be able to test them in queries later.

Reply:
fixed